core/kt-osrd-path/src/main/java/fr/sncf/osrd/path/implementations/EnvelopeSimPath.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Map;
 
 public final class EnvelopeSimPath implements PhysicsPath {
+    // TODO: split that class into interface + implementation, for easier integration in TrainPath
```

Comment:
That *could* be its own issue, and someone else could work on that in parallel. If there's any spare bandwidth around.

Not *strictly* a requirement for backtracks, but it would make the transition easier IMO.